package.json

```diff
@@ -52,11 +52,11 @@
 		"nanoid": "^3.1.23",
 		"react-dropzone": "10.2.0",
 		"react-inlinesvg": "^2.3.0",
-		"react-pdf": "^4.1.0",
+		"react-pdf": "^5.3.2",
```

Comment:
Why did you update this package? Did you check any potential compatibility issues?